resources/measures/upgrade_hvac_vrf_hr_doas/measure.rb

```diff
@@ -1812,118 +1872,201 @@ def run(model, runner, user_arguments)
+          puts("--- #{coil_cooling.name} | design_cooling_load = #{design_cooling_load} W")
+
+          # get design_heating_load
+          column_name = 'Zone Sensible Heat Gain at Ideal Loads Peak'
+          design_heating_load = get_tabular_data(model, sql, 'CoilSizingDetails', 'Entire Facility', 'Coils', row_name_heating, column_name).to_f
+          puts("--- #{coil_cooling.name} | design_heating_load = #{design_heating_load} W")
+
+          # get capacity_original_rated
+          column_name = 'Coil Final Gross Total Capacity'
+          capacity_original_rated = get_tabular_data(model, sql, 'CoilSizingDetails', 'Entire Facility', 'Coils', row_name_cooling, column_name).to_f
+          puts("--- #{coil_cooling.name} | capacity_original_rated = #{capacity_original_rated} W")
+
+          # skip upsizing if indoor unit is not expected as heating dominant unit
+          if design_heating_load <= design_cooling_load
+            puts("--- #{coil_cooling.name} | this indoor unit is not expected as heating dominant, so skipping for upsizing.")
+            capacity_outdoor_unit_new += capacity_original_rated
+            next
+          end
+
+          # get design_air_flow_rate_m_3_per_sec
+          column_name = 'Coil Air Volume Flow Rate at Ideal Loads Peak'
+          design_air_flow_rate_m_3_per_sec = get_tabular_data(model, sql, 'CoilSizingDetails', 'Entire Facility', 'Coils', row_name_cooling, column_name).to_f
+          puts("--- #{coil_cooling.name} | design_air_flow_rate_m_3_per_sec = #{design_air_flow_rate_m_3_per_sec} m3/s")
+
+          # get fan_heat_gain
+          column_name = 'Supply Fan Air Heat Gain at Ideal Loads Peak'
+          fan_heat_gain = get_tabular_data(model, sql, 'CoilSizingDetails', 'Entire Facility', 'Coils', row_name_cooling, column_name).to_f
+          puts("--- #{coil_cooling.name} | fan_heat_gain = #{fan_heat_gain} W")
+    
+          # get rated_capacity_modifier
+          column_name = 'Coil Off-Rating Capacity Modifier at Ideal Loads Peak'
+          rated_capacity_modifier = get_tabular_data(model, sql, 'CoilSizingDetails', 'Entire Facility', 'Coils', row_name_cooling, column_name).to_f
+          puts("--- #{coil_cooling.name} | rated_capacity_modifier = #{rated_capacity_modifier}")
+
+          # get capacity_upsized_rated
+          capacity_upsized_rated = capacity_original_rated * (1 + upsizing_allowance_pct / 100.0)
+          puts("--- #{coil_cooling.name} | capacity_upsized_rated = #{capacity_upsized_rated} W")
+
+          # get design capacity from upsized rated capacity
+          capacity_upsized_design = capacity_upsized_rated * rated_capacity_modifier
+          puts("--- #{coil_cooling.name} | capacity_upsized_design = #{capacity_upsized_design}")
+          capacity_upsized_design_wo_fan_heat_gain = capacity_upsized_design - fan_heat_gain
```

Comment:
Why do we need to remove the fan waste heat?

Reply:
based on the definition of design load in EP